spec/controllers/bundle_context_controller_spec.rb

```diff
@@ -1,71 +1,44 @@
 RSpec.describe BundleContextController, type: :controller do
-  context 'GET index' do
-    let(:bc) do
-      BundleContext.new(
```

Comment:
this `let` was never used in this block, and if it had been used, would have crashed because `user` is undefined!